starport/internal/version/version.go

```diff
@@ -59,15 +67,51 @@ func CheckNext(ctx context.Context) (isAvailable bool, version string, err error
 }
 
 // Long generates a detailed version info.
-func Long() string {
-	output := fmt.Sprintf("starport version %s %s/%s -build date: %s",
-		Version,
-		runtime.GOOS,
-		runtime.GOARCH,
-		Date)
-
-	if Head != "" {
-		output = fmt.Sprintf("%s\ngit object hash: %s", output, Head)
+func Long(ctx context.Context) string {
+	var (
+		w = &tabwriter.Writer{}
+		b = &bytes.Buffer{}
+	)
+
+	write := func(k string, v interface{}) {
+		fmt.Fprintf(w, "%s:\t%v\n", k, v)
 	}
-	return output
+
+	w.Init(b, 0, 8, 0, '\t', 0)
+
+	write("Starport version", Version)
+	write("Starport build date", Date)
+	write("Starport source hash", Head)
+
+	write("Your os", runtime.GOOS)
+	write("Your arch", runtime.GOARCH)
+
+	cmdOut := &bytes.Buffer{}
+
+	err := exec.Exec(ctx, []string{"go", "version"}, exec.StepOption(step.Stdout(cmdOut)))
+	if err != nil {
+		panic(err)
+	}
+	write("Your go version", strings.TrimSpace(cmdOut.String()))
+
```

Comment:
```suggestion
	write("Starport version", Version)
	write("Starport build date", Date)
	write("Starport source hash", Head)

	write("Your OS", runtime.GOOS)
	write("Your arch", runtime.GOARCH)

	cmdOut := &bytes.Buffer{}

	err := exec.Exec(ctx, []string{"go", "version"}, exec.StepOption(step.Stdout(cmdOut)))
	if err != nil {
		panic(err)
	}
	write("Your Go version", strings.TrimSpace(cmdOut.String()))

```

Reply:
Capitalize OS and Go

Reply:
Great, thanks!

Reply:
I capitalized OS. `go version` stays for an actual command name so it's better to keep it lowecase.